jvmti-access/src/main/jni/ProfilerSocket.cpp

```diff
@@ -3,6 +3,7 @@
 #include <sys/socket.h>
 #include <sys/un.h>
 #include <fcntl.h>
+#include <cstring>
```

Comment:
is it required to add it ? does not seem used unless there is something implicit.

Reply:
Yup, it is required. Looks like with glibc it was included transitively form one of the other headers, but not for musl